src/core/trulens/core/session.py

```diff
@@ -176,6 +179,170 @@ def __init__(self, connector: Optional[DBConnector] = None, **kwargs):
             **self_args,
         )
 
+    def App(self, *args, app: Optional[Any] = None, **kwargs) -> App:
+        """Create an App from the given App constructor arguments by guessing
+        which app type they refer to.
+
+        This method intentionally prints out the type of app being created to
+        let user know in case the guess is wrong.
+        """
+        if app is None:
+            # If app is not given as a keyword argument, check the positional args.
+
+            if len(args) == 0:
+                # Basic app can be specified using the text_to_text key argument.
+                if "text_to_text" in kwargs:
+                    from trulens.core.app import basic
+
+                    return basic.TruBasicApp(
+                        *args, connector=self.connector, **kwargs
+                    )
+
+                raise ValueError("No app provided.")
+
+            # Otherwise the app must be the first positional arg.
+            app, args = args[0], args[1:]
+
+        # Check for optional app types.
+        if app.__module__.startswith("langchain"):
+            with import_utils.optional_import(
```

Comment:
I assume you mean `import_utils.OptionalImports` here

Reply:
yes